src/saving/InProgressLoad.cs

```diff
@@ -13,6 +13,7 @@
 /// </remarks>
 public class InProgressLoad
 {
+    public static bool isLoading { get; private set; }
```

Comment:
Add a blank line after this, and also add a documentation comment on this for what this is used for and when it is true.

Reply:
Also as the CI correctly detected this should be named IsLoading